README.md

```diff
@@ -62,7 +89,41 @@ router.get('/', function (req, res) {
 });
 ```
 
-As it turns out, this middleware will do the above by default, so if you don't pass a store you will automagically get all of the properties from `res.locals` passed as props to your component.
+If passing a single container, this middleware will do the above by default, so if you don't pass a store you will automagically get all of the properties from `res.locals` passed as props to your component.
+
+If you pass jsx, you gain flexibility, but lose the 'automagical' convenience described above, and must pass props down directly.
+
+### Options
+
+```javascript
+// *
+// Generating middleware with config options
+// *
+router.use(reactExpressMiddleware({
+	 // [ Options and their defaults ]
+	 // -- Client
+		 // element: (defaults to ‘body’)
+		 // clientRenderMethod: (defaults to ReactDOM.render)
+	 // -- Server
+		 // template: (defaults to ‘index’)
+		 // key: (defaults to ‘context’)
+		 // serverRenderMethod: (defaults to ReactDOMServer.renderToString)
+})
+
+// *
+// Using ‘renderReactComponent’ method.
+// *
+res.renderReactComponent(Component, store, done)
```

Comment:
I think something more (akin to traditional api docs)[https://github.com/wesleytodd/runnable#api] would be good for this then:

```
runnable(fnc <Function>[,defaults <Array>[, module <Object>]]);
```

Have you seen this format before?  the `[]` indicate optional arguments, and the `<>` indicate the option type.  So it would be like:

```
res.renderReactComponent(Component <ReactComponentConstructor>[, store <Object>[, done <Function>]])
res.renderReactComponent(component <ReactComponent>[, done <Function>])
```